gammapy/data/data_store.py

```diff
@@ -711,3 +713,47 @@ def make_psf(self, position, energy=None, theta=None):
         psf_tot = EnergyDependentTablePSF(energy=energy, offset=theta, exposure=exposure,
                                           psf_value=psf_value.T)
         return psf_tot
+
+    def make_mean_edisp(self, position, e_true, e_reco, low_reco_threshold=Energy(0.002, "TeV"),
+                        high_reco_threshold=Energy(150, "TeV")):
+        r"""Make mean edisp for a given position and a set of observations.
+
+        Compute the mean edisp of a set of observations j at a given position
+
+        The stacking is implemented in :func:`~gammapy.irf.IRFStacker.stack_edisp
+
+        Parameters
+        ----------
+        position : `~astropy.coordinates.SkyCoord`
+            Position at which to compute the mean EDISP
+        e_true : `~gammapy.utils.energy.EnergyBounds`
+            True energy axis
+        e_reco : `~gammapy.utils.energy.EnergyBounds`
+            Reconstructed energy axis
+        low_reco_threshold : `~gammapy.utils.energy.Energy`
+            low energy threshold in reco energy, default 0.002 TeV
+        high_reco_threshold : `~gammapy.utils.energy.Energy`
+            high energy threshold in reco energy , default 150 TeV
+
+        Returns
+        -------
+        stacked_edisp : `~gammapy.irf.EnergyDispersion`
+            Stacked EDISP for a set of observation
+        """
+
+        list_aeff = list()
+        list_edisp = list()
+        list_livetime = list()
+        list_low_threshold = [low_reco_threshold] * len(self)
```

Comment:
@joleroi 
needed in the method stack_edisp since you can have e_reco thresholds different for each observation in your spectral extraction.
Here for the moment I don't know what we have to use for thresholds so if a user want to put one he can!!